stem/utils.py

```diff
@@ -376,6 +376,47 @@ def create_sigmoid_tiny_expr(start_time: float, dt_slope: float, initial_value:
             return (f"(1 / (1 + e^(-{beta} * (t - {dt_slope} / 2 - {start_time})))) * ({final_value} - {initial_value})"
                     f"+ {initial_value}")
 
+    @staticmethod
+    def create_box_tiny_expr(transition_parameter: float,
+                             start_peak: float,
+                             end_peak: float,
+                             peak_value: float,
+                             base_value: float,
+                             variable: str = "x") -> str:
+        """
+        Creates a tiny expression for a box function. For more information on tiny expressions,
+        see: https://github.com/codeplea/tinyexpr
+
+        Args:
+            - transition_parameter (float): parameter to control the transition of the box function, |
+              the higher the value, the steeper the transition
+            - start_peak (float): start of the peak of the box function
+            - end_peak (float): end of the peak of the box function
+            - peak_value (float): value of the peak of the box function
+            - base_value (float): value of the base of the box function
+            - variable (str): variable within the box function tinyexpr, default is "x", other options are "y", "z", "t"
+
+        Raises:
+            - ValueError: when start peak is larger or equal to end peak
+
+        Returns:
+            - str: tiny expression of the box function
+
+        """
+
+        if start_peak >= end_peak:
+            raise ValueError("Start peak should be smaller than end peak.")
+
```

Comment:
add a ValueError for variable: 
if variable not in ['x', 'y', 'z', 't']:
    raise ValueError(".....")